docs/docs/getting-started/integration.md

```diff
@@ -80,6 +80,13 @@ Chart.register(
 var myChart = new Chart(ctx, {...});
 ```
 
+A short registration format is also available to quickly register everything.
+
+```javascript
+import { Chart, registerables } from 'chart.js';
+Chart.register(...registerables);
```

Comment:
I'm a bit annoyed about the need to destructuring.
Would it work without it, if the export was:
`export const registrables = Object.assign({}, controller, elements, plugins, scales)`? (not sure if this would break tree shaking)

Reply:
Tree shaking isn't working with that unfortunately. The final built production bundle has this code in it. It builds the `registerables` content, but never assigns it. This ends up keeping all of the code in, even though I only imported what was needed

```javascript
var So = Object.freeze({
    __proto__: null,
    CategoryScale: eo,
    LinearScale: oo,
    LogarithmicScale: ao,
    RadialLinearScale: fo,
    TimeScale: wo,
    TimeSeriesScale: Po
  });
  Object.assign({
  }, Xe, vn, to, So),
```

Reply:
ok. another thought (no need to pursue this way)

registrables.js:
```js
export * as controllers from './controllers';
export * as elements from './elements';
export * as plugins from './plugins';
export * as scales from './scales';
```

index.esm.js:
```js
export * as registrables from './registrables';
```

Reply:
`registerables` would be an object so I don't think it'd work as expected